infrastructure/kzg/src/test/java/tech/pegasys/teku/kzg/KZGTest.java

```diff
@@ -40,27 +41,21 @@ public final class KZGTest {
   private static final int RANDOM_SEED = 5566;
   private static final Random RND = new Random(RANDOM_SEED);
 
-  @BeforeEach
-  public void setup() {
-    KZG.resetTrustedSetup();
-  }
-
-  @Test
-  public void testKzgDoubleResetDoNotThrow() {
-    KZG.resetTrustedSetup();
```

Comment:
If we keep exception for `resetTrustedSetup`, I'd test it, just change name to `testKzgDoubleResetThrows`
```
    loadTrustedSetup();
    KZG.resetTrustedSetup();
    assertThrows(KZGException.class, () -> KZG.resetTrustedSetup());
```